tck/src/main/java/ee/jakarta/tck/data/framework/read/only/AsciiCharacters.java

```diff
@@ -70,11 +71,17 @@ Stream<AsciiCharacter> findByHexadecimalIgnoreCaseBetweenAndHexadecimalNotIn(Str
 
     Optional<AsciiCharacter> findFirstByHexadecimalStartsWithAndIsControlOrderByIdAsc(String firstHexDigit, boolean isControlChar);
 
+    @Query("WHERE hexadecimal <> ' ORDER BY isn''t a keyword when inside a literal' AND hexadecimal IN ('4a', '4b', '4c', ?1)")
```

Comment:
If it is verified at build time, it might not compile.

Reply:
> If it is verified at build time, it might not compile.

Can you explain why and where the error is? As best I can see, the query language used is consistent with what is defined in the spec here. A user can write this query and they will expect it to work, per the spec.